Do not run python tests, they take eternity to finish.

--- a/wscript
+++ b/wscript
@@ -370,21 +370,13 @@
     ret = samba_utils.RUN_COMMAND(cmd)
     print("testsuite returned %d" % ret)
 
-    tmp_dir = os.path.join(test_prefix, 'tmp')
-    if not os.path.exists(tmp_dir):
-        os.mkdir(tmp_dir)
-    pyret = samba_utils.RUN_PYTHON_TESTS(
-        ['tests/python/api.py', 'tests/python/index.py'],
-        extra_env={'SELFTEST_PREFIX': test_prefix})
-    print("Python testsuite returned %d" % pyret)
-
     cmocka_ret = 0
     for test_exe in ['ldb_tdb_mod_op_test',
                      'ldb_msg_test']:
             cmd = os.path.join(Utils.g_module.blddir, test_exe)
             cmocka_ret = cmocka_ret or samba_utils.RUN_COMMAND(cmd)
 
-    sys.exit(ret or pyret or cmocka_ret)
+    sys.exit(ret or cmocka_ret)
 
 def dist():
     '''makes a tarball for distribution'''
